pkg/odo/util/pushtarget/pushtarget_test.go

```diff
@@ -0,0 +1,114 @@
+package pushtarget
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/openshift/odo/pkg/preference"
+)
+
+func TestIsDockerPushTargetEnabled(t *testing.T) {
```

Comment:
Please change the name to the corresponding function it tests `PushTargetDocker`.

Reply:
Updated.